src/Core/Configuration/Configuration.php

```diff
@@ -195,7 +195,7 @@ public function getCacheDir(): ?string
         } catch (ObjectNotFoundException) {
         }
 
-        $cacheDir = $this->parameterBag->validateAndGetStringValue('cache_dir');
+        $cacheDir = $this->parameterBag->validateAndGetStringValue(ConfigurationKey::CACHE_DIR);
         $parentDir = dirname($cacheDir);
         if (!is_dir($parentDir)) {
             throw new InvalidConfigurationParameterException(
```

Comment:
Can you change `cache_dir` in the message here to `ConfigurationKey::CACHE_DIR`?